src/test/java/plugins/MatrixAuthPluginTest.java

```diff
@@ -48,14 +49,23 @@ public void helloWorld() throws Exception {
         jenkins.login().doLogin("alice");
 
         FreeStyleJob j = jenkins.jobs.create();
+
         j.save();
 
         // if we login as Bob, he shouldn't see the job
         jenkins.login().doLogin("bob");
+
+        //wait for main panel to appear to make sure page is rendered
+        waitFor(by.id("main-panel"));
+
         assertNull(getElement(by.href("job/"+j.name+"/")));
 
         // contorl assertion: alice shoudl see the link
         jenkins.login().doLogin("alice");
+
+        //wait for main panel to appear to make sure page is rendered
+        waitFor(by.id("main-panel"));
```

Comment:
@amuniz For that reason, nobody changes things that should be changed. In fact, this `ID` will be changed soon by me. There are other alternatives to do the check. For example, checking if the HTTP request has finished (page loaded)